sale_promotion_discount_in_field/tests/test_promotion_discount_in_field.py

```diff
@@ -0,0 +1,228 @@
+from odoo.tests import tagged
+from odoo.tests.common import SavepointCase
+
+
+@tagged("post_install", "-at_install")
+class CetmixTestSaleCouponProgram(SavepointCase):
+    def setUp(self):
```

Comment:
pls rewrite this to use `setUpClass` and set tracking_disable.

Example: https://github.com/OCA/wms/blob/14.0/shopfloor_base/tests/common.py#L99-L108